workflows/iuf/operations/vcs-upload/vcs-upload-content.yaml

```diff
@@ -227,7 +227,7 @@ spec:
               path: /results/records.yaml
               default: "{}"
       container:
-        image: registry.local/artifactory.algol60.net/csm-docker/stable/cf-gitea-import:1.8.0-alpha.19_6747be7
+        image: registry.local/artifactory.algol60.net/csm-docker/stable/cf-gitea-import:1.8.1
```

Comment:
This should already be bumped in main, merged yesterday

Reply:
Ah, I'll resolve the merge conflict.